fe/fe-core/src/main/java/org/apache/doris/statistics/ColumnStats.java

```diff
@@ -72,16 +75,94 @@ public class ColumnStats {
     private float avgSize = -1;  // in bytes
     private long maxSize = -1;  // in bytes
     private long numNulls = -1;
-    private LiteralExpr minValue;
-    private LiteralExpr maxValue;
-
-    public static ColumnStats createDefaultColumnStats() {
-        ColumnStats columnStats = new ColumnStats();
-        columnStats.setAvgSize(1);
-        columnStats.setMaxSize(1);
-        columnStats.setNdv(1);
-        columnStats.setNumNulls(0);
-        return columnStats;
+
+    private LiteralExpr minValue = new NullLiteral();
+    private LiteralExpr maxValue = new NullLiteral();
+
+    /**
+     * Return default column statistic.
+     */
+    public static ColumnStats getDefaultColumnStats() {
+        return new ColumnStats();
+    }
+
+    /**
+     * Merge column statistics(the original statistics should not be modified)
+     *
+     * @param left statistics to be merged
+     * @param right statistics to be merged
+     */
+    public static ColumnStats aggColumnStats(ColumnStats left, ColumnStats right) {
```

Comment:
rename this method to `mergeColumnStats`?

Reply:
ok